src/java/org/apache/cassandra/service/paxos/Ballot.java

```diff
@@ -38,6 +38,7 @@
 public class Ballot extends TimeUUID
 {
     public static final long serialVersionUID = 1L;
+    private static final long LEGACY_MIN_CLOCK_SEQ_AND_NODE = 0x8080808080808080L;
```

Comment:
Do you think it'd be unreasonable to make `MIN_CLOCK_SEQ_AND_NODE` public on `TimeUUID`? If so, maybe adding a small comment about the source of this constant could be useful for context.

Reply:
no I think that would be fine. I'd thought it had been removed from TimeUUID from some reason, which is why I copied it here